packages/dapi-grpc/protos/platform/v0/platform.proto

```diff
@@ -2,20 +2,25 @@ syntax = "proto3";
 
 package org.dash.platform.dapi.v0;
 
-import "google/protobuf/timestamp.proto";
-
 service Platform {
-  rpc broadcastStateTransition (BroadcastStateTransitionRequest) returns (BroadcastStateTransitionResponse);
-  rpc getIdentity (GetIdentityRequest) returns (GetIdentityResponse);
-  rpc getDataContract (GetDataContractRequest) returns (GetDataContractResponse);
-  rpc getDocuments (GetDocumentsRequest) returns (GetDocumentsResponse);
-  rpc getIdentitiesByPublicKeyHashes (GetIdentitiesByPublicKeyHashesRequest) returns (GetIdentitiesByPublicKeyHashesResponse);
-  rpc waitForStateTransitionResult (WaitForStateTransitionResultRequest) returns (WaitForStateTransitionResultResponse);
-  rpc getConsensusParams (GetConsensusParamsRequest) returns (GetConsensusParamsResponse);
-}
-
-message Proof {
-  bytes merkle_proof = 1;
+  rpc broadcastStateTransition(BroadcastStateTransitionRequest)
+      returns (BroadcastStateTransitionResponse);
+  rpc getIdentity(GetSingleItemRequest) returns (SingleItemResponse);
```

Comment:
I'm afraid that we can not change `.proto` specification at this point it time. 
It will affect many things through the whole **client** -> **server** chain, and this is not something we can deal with efficiently until `platform-test-suite` is stabilized. 

cc @QuantumExplorer